provider/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -527,7 +524,6 @@ func resourceContainerCluster() *schema.Resource {
 				ForceNew:   true,
```

Comment:
On lines 514'ish we should clean up the deprecation logic

Reply:
Done.